src/xrGame/WeaponMagazined.cpp

```diff
@@ -98,7 +98,10 @@ void CWeaponMagazined::Load(LPCSTR section)
             m_sSilencerSmokeParticles = pSettings->r_string(section, "silencer_smoke_particles");
 
         //Alundaio: LAYERED_SND_SHOOT Silencer
-        m_layered_sounds.LoadSound(section, "snd_silncer_shot", "sndSilencerShot", false, m_eSoundShot);
+        if (!m_layered_sounds.LoadSoundTry(section, "snd_silncer_shot", "sndSilencerShot", false, m_eSoundShot))
+        {
+          m_layered_sounds.LoadSound(section, "snd_shoot", "sndSilencerShot", false, m_eSoundShot);
+        }
```

Comment:
The existing pipeline when processing LTX is to explicitly check for line existence is to use `line_exist` function.
There are helpers like `read_if_exists` functions and `READ_IF_EXISTS` macro, they should be used when we can assign the value immediately and straightforward.

For weapon sounds we have a helper function `WeaponSoundExist`, please use it, don't duplicate existing functionality.
Also, is it possible to assign already loaded sound collection (`sndShot`) to the one we are processing here (`sndSilencerShot`)? It would be optimal to do this.